modules/ting/src/Search/TingSearchResultInterface.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * @file
+ * The TingSearchResultInterface interface.
+ */
+
+namespace Ting\Search;
+
+/**
+ * Interface TingSearchResultInterface
+ *
+ * @package Ting\Search
+ */
+interface TingSearchResultInterface {
```

Comment:
I would like things to settle a bit more before doing the renaming - I agree that the current naming of  the count-functions is too messy. I'll add a note that it has to be considered.